Packs/FeedDHS/Integrations/DHSFeedV2/DHSFeedV2.py

```diff
@@ -0,0 +1,223 @@
+    :param limit: upper limit of indicators to fetch
+    :param last_run_ctx: last run dict with {collection_id: last_run_time string}
+    :param initial_interval: initial interval in human readable format
+    :param fetch_full_feed: when set to true, will ignore last run, and try to fetch the entire feed
+    :return: indicators in cortex TIM format
+    """
+    initial_interval = dateparser.parse(initial_interval, date_formats=[TAXII_TIME_FORMAT])
+
+    if client.collection_to_fetch:
+        indicators, last_run_ctx = fetch_one_collection(client, limit, last_run_ctx, initial_interval, fetch_full_feed)
+    else:
+        indicators, last_run_ctx = fetch_all_collections(client, limit, initial_interval, last_run_ctx, fetch_full_feed)
+
+    return indicators, last_run_ctx
+
+
+def fetch_one_collection(client: Taxii2FeedClient, limit: int, last_run_ctx: dict, initial_interval: str, fetch_full_feed: bool):
+    last_fetch_time = last_run_ctx.get(client.collection_to_fetch.id)
+    added_after = get_added_after(fetch_full_feed, initial_interval, last_fetch_time)
+
+    indicators = client.build_iterator(limit, added_after=added_after)
+    last_run_ctx[client.collection_to_fetch.id] = (client.last_fetched_indicator__modified
+                                                   if client.last_fetched_indicator__modified
+                                                   else added_after)
+
+    return indicators, last_run_ctx
+
+
+def fetch_all_collections(client: Taxii2FeedClient, limit: int, initial_interval: str, last_run_ctx: Optional[dict] = None,
+                          fetch_full_feed: bool = False):
+    indicators: list = []
+    for collection in client.collections:
+        client.collection_to_fetch = collection
+        added_after = get_added_after(fetch_full_feed, initial_interval,
+                                      last_run_ctx.get(collection.id) if last_run_ctx else None)
+        fetched_iocs = client.build_iterator(limit, added_after=added_after)
+        indicators.extend(fetched_iocs)
+        if last_run_ctx:
+            last_run_ctx[collection.id] = client.last_fetched_indicator__modified
+
+        if limit >= 0:
+            limit -= len(fetched_iocs)
+            if limit <= 0:
+                break
+
+    return indicators, last_run_ctx
+
+
+def get_added_after(fetch_full_feed: bool, initial_interval: Union[str, datetime.datetime], last_fetch_time: str = None):
+    if fetch_full_feed:
+        return initial_interval
+
+    return last_fetch_time or initial_interval
+
+
+def get_indicators_command(client: Taxii2FeedClient, raw: str = 'false', limit: str = '10', added_after: str = '20 days'):
+    """
+    Fetch indicators from TAXII 2 server
+    :param client: Taxii2FeedClient
+    :param raw: When set to 'true' will return only rawJSON
+    :param limit: upper limit of indicators to fetch
+    :param (Optional) added_after: added after time string in parse_date_range format
```

Comment:
It's not really optional
```suggestion
    :param added_after: added after time string in parse_date_range format
```